ios/MullvadVPN/PacketTunnelIpc.swift

```diff
@@ -6,43 +6,38 @@
 //  Copyright © 2019 Mullvad VPN AB. All rights reserved.
 //
 
-import Combine
 import Foundation
 import NetworkExtension
 
 /// A enum describing the kinds of requests that `PacketTunnelProvider` handles
-enum PacketTunnelRequest: Int, Codable {
-    /// Request the tunnel to reload the configuration
-    case reloadConfiguration
+enum PacketTunnelRequest: Int, Codable, RawRepresentable {
```

Comment:
On iOS 13, JSONEncoder is able to serialise a single primitive into JSON, however on iOS 12 it throws an error saying that it needs a top level object to encode it as JSON. So basically I changed this `enum` to encode itself into the dictionary of following shape:

```
{ type: Int }
```